app/src/main/res/layout/activity_main.xml

```diff
@@ -113,27 +112,28 @@
                         app:srcCompat="@mipmap/no_messages" />
 
                     <TextView
-                        android:id="@+id/emptyMessageText"
+                        android:id="@+id/message_list_loading_text"
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
                         android:layout_marginTop="@dimen/meshim_margin_small"
                         android:text="@string/empty_message_text1"
                         android:textStyle="bold"
                         app:layout_constraintEnd_toEndOf="parent"
                         app:layout_constraintStart_toStartOf="parent"
-                        app:layout_constraintTop_toBottomOf="@+id/emptyMessageImage" />
+                        app:layout_constraintTop_toBottomOf="@+id/message_list_loading_image" />
 
                     <TextView
-                        android:id="@+id/emptyMessageText1"
+                        android:id="@+id/message_list_loading_text1"
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
                         android:layout_marginEnd="@dimen/meshim_margin_small"
                         android:layout_marginStart="@dimen/meshim_margin_small"
-                        android:layout_marginTop="@dimen/meshim_margin_small"
+                        android:layout_marginTop="8dp"
```

Comment:
WYSIWYG editor snuck a magic number in again.

Reply:
fixed it!